<div class="container-fluid">
  <div class="row">
    <div class="mx-auto col-11 col-sm-10 col-lg-8 col-xl-6">
      <h3 class="mt-4">ユーザー編集</h3>
    </div>
  </div>
  <div class="row py-5 mx-auto mt-4 shadow-lg w-50 d-flex align-items-center justify-content-center">
    <div class="col-10">
      <div class="row d-flex align-items-center justify-content-center">
        <div class="col-8 d-flex justify-content-center px-0">
          <% if @user.errors.any? %>
            <%= @user.errors.count %>件のエラーが発生しました
            <ul>
              <% @user.errors.full_messages.each do |message| %>
                <li><%= message %></li>
              <% end %>
            </ul>
          <% end %>
        </div>
        <%= form_with model: @user do |f| %>
          <div class="col-9 mt-4">
            <h4 class="mt-3">ユーザーネーム</h4>
            <%= f.text_field :name, id:"inputName" %>
          </div>
          <div class="col-9 mt-4">
            <h4 class="mt-3">プロフィール画像</h4>
            <%= f.file_field :profile_image, accept: "image/*" %>
          </div>
          <div class="col-9 mt-4 d-flex align-items-center justify-content-center">
            <%= f.submit "変更", class: "btn btn-info mr-4" %>
            <%= link_to "退会", users_check_path, class: "btn btn-danger" %>
          </div>
        <% end %>
      </div>
      <div class="col-12 d-flex align-items-center justify-content-end">
        <%= link_to "BACK", user_path(current_user.id) %>
      </div>
    </div>
  </div>
</div>